### DISCUSSION TITLE
I have some questions pls

### DISCUSSION THREAD

**Discussion by Ameryah** [Chia-Network/chia-blockchain#12308] [Support]
Hey !
I have several questions to ask if you would like to answer them it would help me a lot !!!

1. I finished installing the mining farm now it says: "syncing" in status. I'm waiting I'm waiting I wanted to know how long it would take approximately

2. what is the minimum configuration to mine chia

I have an acer nitro:
cpu: i-5 10300H 8 threads 4 cores
ram: 2 x 4gb 2933Mhz
gpu: rtx 3060 VRAM6go
Storage: 512GB PCIe NVMe SSD

and "my passport ultra 1tb" for mining

3. for "Choose Number of Plots"
if I choose to put 3 plots
will the size of the temporary file be multiplied by 3 too?
or just the final file so in my "passport ultra 1tb"

**Comment by loppefaaret** [comment]
to create a k32 plot, which takes up just over 100GiB - you need about 260GiB of temp space while creating the plot, using our chiaPOS plotter
as for min requirements, some people are successfully farming with a raspberry Pi 4 - i would not plot on such a device though... but generally, quite lowpowered devices should be able to keep up, as long as they have a SSD based device to work on
